src/item.h

```diff
@@ -98,7 +98,7 @@ class item : public JsonSerializer, public JsonDeserializer
 {
 public:
  item();
- item(const std::string new_type, unsigned int turn, bool rand = true, handedness handed = NONE);
+ item(const std::string new_type, int turn, bool rand = true, handedness handed = NONE);
```

Comment:
This doesn't look right, `item` is often called with either `0` as turn or `calendar::turn`, which is a calendar object, which has a `operator int()`. In both cases the int (`0`)  or the int from `calendar` is "converted" to unsigned for no actual reason.

Even worse: the turn is stored in `item::bday`, which is a int again, so it's converted back to `int` anyway. Why all that?